app/controllers/users/phones_controller.rb

```diff
@@ -19,98 +19,23 @@ def create
       end
     end
 
-    def edit
-      set_phone_id
-      # memoized for view
-      @edit_phone_form = EditPhoneForm.new(current_user, phone_configuration)
-    end
-
-    def update
-      @edit_phone_form = EditPhoneForm.new(current_user, phone_configuration)
-      if @edit_phone_form.submit(edit_params).success?
-        process_updates
-        bypass_sign_in current_user
-      else
-        render :edit
-      end
-    end
-
-    def delete
-      result = TwoFactorAuthentication::PhoneDeletionForm.new(
-        current_user, phone_configuration
-      ).submit
-      analytics.track_event(Analytics::PHONE_DELETION_REQUESTED, result.to_h)
-      if result.success?
-        handle_successful_delete
-      else
-        flash[:error] = t('two_factor_authentication.phone.delete.failure')
-      end
-
-      redirect_to account_url
-    end
-
     private
 
-    def render_edit
```

Comment:
Should we allow users to add duplicate phone numbers?

Reply:
This method was vestigial. It is never called anywhere.

This is a relic of the edit phone form allowing you to modify a phone number. The functionality was removed and this method was left behind.